python/requirements.txt

```diff
@@ -1,2 +1,4 @@
+numpy==1.21.1
+scipy==1.7.1
```

Comment:
How this came up here?

Reply:
Because of `nxalg` module. [Here](https://memgraph.com/docs/mage/query-modules/python/nxalg)

Reply:
Because of `nxalg` module. [Here](https://memgraph.com/docs/mage/query-modules/python/nxalg)